instructions.cpp

```
@@ -2,31 +2,19 @@
 Team Void: UVSim
 instructions.cpp
 */
-#include <memory.h>
 #include <iostream>
+#include "instructions.h"
+#include "memory.h"
 using namespace std;
 
 
-class instructions {
-    public:
-        void read(int operand);
-        void write(int operand);
-        void load(int operand);
-        void store(int operand);
-        void add(int operand);
-        void subtract(int operand);
-        void divide(int operand);
-        void multiply(int operand);
-    private:
-        bool outOfBounds(int value);
-};
 
 //I/O Operations
 
 void instructions::read(int operand, Memory m){
   int input_value;
   cout<<"\nEnter an integer: ";
-  cin>>input_value;
+  cin>>input_value;//DO we need to validate or call validation method to re-enter data if bad?
```

I think it should throw an error saying that the information is bad. A user should know that one of their instruction was not valid

---

OOTB: We're going to alert when the user does something wrong